lib/discourse_api/api/uploads.rb

```diff
@@ -8,6 +8,14 @@ def upload_post_image(args)
                   .to_h
         post('/uploads', args)
       end
+
+      def upload_file(args)
```

Comment:
`upload_post_image` method name is very specific which shows that it would be used to upload only images. Therefore, I have added a new method instead of renaming the original to ensure backward dependency. Moreover `composer` didnt really work for me when I tried to upload he file.